src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -783,6 +783,19 @@ let setup_daemon logger ~itn_features ~default_snark_worker_fee =
               ~proof_level:Genesis_constants.Compiled.proof_level config_files
               ~genesis_constants ~constraint_constants ~cli_proof_level
           in
+
+          (* TODO consider moving elsewhere *)
+          let duration =
+            constraint_constants.block_window_duration_ms |> Float.of_int
+            |> Time.Span.of_ms
+          in
+          Mina_metrics.Transition_frontier.TPS_30min.initialize duration ;
```

Comment:
Suggestion: 

we should move this to a `Mina_metrics.inititalize_all_the_things : block_window_duration -> unit`  so that we can keep track of this in one place.